src/components/ImageView/index.native.js

```diff
@@ -1,35 +1,44 @@
 import React, {PureComponent} from 'react';
 import PropTypes from 'prop-types';
 import {
-    View, InteractionManager, PanResponder,
+    View, PanResponder, InteractionManager,
```

Comment:
NAB - this change isn't needed

```suggestion
    View, InteractionManager, PanResponder,
```